internal/annotations/metrics/metrics.go

```diff
@@ -46,7 +46,7 @@ const (
 func MakeBundleMetadataLabels(cfg *config.Config) map[string]string {
 	return map[string]string{
 		mediaTypeBundleAnnotation: mediaTypeV1,
-		builderBundleAnnotation:   getSDKBuilder(sdkversion.Version),
+		builderBundleAnnotation:   getSDKBuilder(strings.TrimSuffix(sdkversion.Version, "+git")),
```

Comment:
Same comment as in https://github.com/operator-framework/operator-sdk/pull/4051

> We should talk about this change. If users generate a bundle with a non-tagged binary, we'll think they used the previous tag.

I don't think we should sacrafice the desired functionality of SDK to make our lives easier for CI.

Reply:
@joelanford,

We trim `sdkversion.Version` in all places. Only the in these 2 places  from all that uses the bin version the trim is not made. The change just removes `+git` when/if it is added. Shows that would not make sense use `+git` on it.  

If the `MakeBundleMetadataLabels ` should not use `sdkversion.Version` then, I think it would need to be solved in a follow-up. WDYT?

Reply:
@joelanford 

I am confused now. I revert this change and all worked.  I saw seen it falling because of `+git suffix.`